spec/takeaway_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:burger_description) {"Cheeseburger"}
+  let(:burger_price) {4.99}
+  let(:burger_to_s) {"#{burger_description}\t#{burger_price}"}
+  let(:burger) do double :dish,
+                  description:  burger_description,
+                  price: burger_price,
+                  to_s: burger_to_s
+  end
+  let(:kebab_description) {"Kebab"}
+  let(:kebab_price) {3.00}
+  let(:kebab_to_s) {"#{kebab_description}\t#{kebab_price}"}
+  let(:kebab) do
+    double :dish,
+    description: kebab_description,
+    price: kebab_price,
+    to_s: kebab_to_s
+  end
+  let(:order_lines) {{burger =>  1, kebab =>  1}}
+  let(:menu) {double :menu, to_s: "#{burger_to_s}\n#{kebab_to_s}\n"}
+  let(:menu_klass) {double :menu_klass, new: menu}
+  let(:total) {burger_price + kebab_price}
+  let(:order) {double :order, total: total}
+  let(:order_klass) {double :order_klass, new: order}
+  subject(:takeaway) do described_class.new(menu_klass: menu_klass,
```

Comment:
Style/MultilineBlockLayout: Block body expression is on the same line as the block start.